app/controllers/tool_groups_controller.rb

```diff
@@ -0,0 +1,82 @@
+class ToolGroupsController < ApplicationController
+  before_action :authorize!
+  before_action :convert_hyphen_to_dash, only: [:create, :update]
+
+  def index
+    render json: tool_groups_ordered_by_name, include: params[:include], fields: field_params, status: :ok
+  end
+
+  def create
+    create_tool_group
+  rescue ActiveRecord::RecordInvalid => e
+    render json: {errors: formatted_errors(e)}, status: :unprocessable_entity
+  end
+
+  def create_tool
+    ResourceToolGroup.create!(
+      tool_group_id: params[:tool_group_id],
+      resource_id: params[:data][:attributes]["resource-id"],
+      suggestions_weight: params[:data][:attributes]["suggestions-weight"]
+    )
+
+    tool_group = ToolGroup.find(params[:tool_group_id])
+    response.headers["Location"] = "tool_groups/#{tool_group.id}"
+    render json: tool_group, status: :created
```

Comment:
we actually should be rendering the ResourceToolGroup here, again I'll work up an example for this API endpoint

Reply:
I would expect the response to look like:
```
{
   "id": 1,
   "type": "tool-group-tool",
   "attributes": {
    "suggestions-weight": "0.1"
   },
   "relationships": {
    "tool-group": {
     "data": {
      "id": 1,
      "type": "tool-group"
     }
    },
    "tool" : {
     "data": {
      "id": 3,
      "type": "resource"
     }
    }
   }
```